canister/src/utxo_set.rs

```diff
@@ -512,6 +515,28 @@ struct BlockIngestionStats {
     ins_insert_utxos: u64,
 }
 
+impl BlockIngestionStats {
+    pub fn get_all_labels() -> Vec<String> {
+        vec![
+            String::from("ins_total"),
+            String::from("ins_remove_inputs"),
+            String::from("ins_insert_outputs"),
+            String::from("ins_txids"),
+            String::from("ins_insert_utxos"),
+        ]
+    }
+
+    pub fn get_labels_and_values(&self) -> Vec<(String, u64)> {
+        vec![
+            (String::from("ins_total"), self.ins_total),
```

Comment:
@ielashi I think `ins_total` is unnecessary because I guess Prometheus will also calculate the sum of all types. What do you think?